src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
+    """
+    The reduction that sometimes is used to lodge the substrate in the substrate holder position..
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    insert_id = Quantity(
+        type=str,
+        description="""
+        The ID of the insert that is placed in this position to accomodate the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    inner_geometry = SubSection(
+        section_def=Geometry,
+    )
+    outer_geometry = SubSection(
+        section_def=Geometry,
+    )
+
+class SubstrateHolderPosition(ArchiveSection):
+    """
+    One casing position of the substrate holder.
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this position.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    slot_geometry = SubSection(
+        section_def=Geometry,
+    )
+    insert_reduction = SubSection(
+        section_def=InsertReduction,
+    )
```

Comment:
Maybe we should add a description saying that this is optional. Something like:
```suggestion
    insert_reduction = SubSection(
        section_def=InsertReduction,
        description='Optional description of insert if used.',
    )
```